docs/DeveloperGuide.md

```diff
@@ -154,6 +154,41 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Delete Meeting Feature
+
+Syntax: `delMeeting i/1`
+
+Purpose: Delete a specified `meeting` from the Meeting List in `Model`
```

Comment:
Maybe caps the `meeting` as a form of standardization?

Reply:
might also need to be more specific about the meeting list in model, bc NoConflictMeetingList is technically in MyInsuRec, which is nested in the ModelManager.

Reply:
Will change it! thanks